### ISSUE TITLE
not working for 2.5.5 internal MSP account

### ISSUE THREAD

**Issue by k1z2** [aruba/central-automation-studio#2]
is this working for 2.5.5 MSP? when I tried to click connect, it showed "can not connect API server"

**Comment by wifidownunder** [comment]
Kevin, I've just tried an MSP account and it is working fine. Make sure that the Local API Proxy is has a green status dot in the footer of any page.

**Comment by k1z2** [comment]
Hi Aaron,  I tried to use the public one and the one I installed on my laptop. But they got the same problem(Internal server error). See screenshot below   BTW the proxy issue is fixed, I can not use localhost but the real ip.   Sent from Mail for Windows From: Aaron ScottSent: Thursday, March 24, 2022 3:33 PMTo: aruba/central-automation-studioCc: kevinzhu74; AuthorSubject: Re: [aruba/central-automation-studio] not working for 2.5.5 internal MSP account (Issue #2) Kevin, I've just tried an MSP account and it is working fine. Make sure that the Local API Proxy is has a green status dot in the footer of any page.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by k1z2** [comment]
Never mind, I forgot to save it to go. I will let you know any other problem This is so great tool! man Sent from Mail for Windows From: Kevin ZhuSent: Thursday, March 24, 2022 9:29 PMTo: aruba/central-automation-studio; aruba/central-automation-studioCc: Author; Zhu, KevinSubject: RE: [aruba/central-automation-studio] not working for 2.5.5 internal MSP account (Issue #2) Hi Aaron, I tried to use the public one and the one I installed on my laptop. But they got the same problem(Internal server error). See screenshot below   BTW the proxy issue is fixed, I can not use localhost but the real ip.   Sent from Mail for Windows From: Aaron ScottSent: Thursday, March 24, 2022 3:33 PMTo: aruba/central-automation-studioCc: kevinzhu74; AuthorSubject: Re: [aruba/central-automation-studio] not working for 2.5.5 internal MSP account (Issue #2) Kevin, I've just tried an MSP account and it is working fine. Make sure that the Local API Proxy is has a green status dot in the footer of any page.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
